src/pages/home/report/ReportActionCompose/ReportActionCompose.js

```diff
@@ -342,120 +342,122 @@ function ReportActionCompose({
     }, [isSendDisabled, resetFullComposerSize, submitForm, animatedRef, isReportReadyForDisplay]);
 
     return (
-        <View
-            ref={containerRef}
-            style={[shouldShowReportRecipientLocalTime && !lodashGet(network, 'isOffline') && styles.chatItemComposeWithFirstRow, isComposerFullSize && styles.chatItemFullComposeRow]}
-        >
-            <PortalHost name="suggestions" />
-            <OfflineWithFeedback
-                pendingAction={pendingAction}
-                style={isComposerFullSize ? styles.chatItemFullComposeRow : {}}
-                contentContainerStyle={isComposerFullSize ? styles.flex1 : {}}
-            >
+        <View style={[shouldShowReportRecipientLocalTime && !lodashGet(network, 'isOffline') && styles.chatItemComposeWithFirstRow, isComposerFullSize && styles.chatItemFullComposeRow]}>
+            <OfflineWithFeedback pendingAction={pendingAction}>
                 {shouldShowReportRecipientLocalTime && hasReportRecipient && <ParticipantLocalTime participant={reportRecipient} />}
```

Comment:
Does this view really require offline feedback?

Reply:
As in the current implementation, it is grayed out this View when a new chat opened in the offline mode, I have kept it bcoz I am not sure about the removing of that.